codegen/safe-eval.ts

```diff
@@ -2,6 +2,28 @@
  *  Copyright (c) Microsoft Corporation. All rights reserved.
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
+import * as vm from 'vm';
 
 /* eslint-disable */
-export const safeEval: <T>(expression: string, context?: any) => T = require('safe-eval');
+
+const sandbox = vm.createContext({});
```

Comment:
Would it make more sense to be able to wrap an instance of a `safeEval` method using another function like `createSafeEval` so that there's not a globally-stored sandbox?  If we use this in other places things might get messy.

Reply:
Also, I think we might need a ThirdPartyNotices file to mention the original MIT license and copyright since we're copying the bulk of the source.

Reply:
Hmm. May be a good point.

Lemme see if I can do that without changing the exported signature.